### PR TITLE
Separate actions menu into 3 different buttons

### PR DESCRIPTION
**Pull request by evmiguel**
This PR replaces the kebab menu into three separate buttons in the Testers column, Report Status button, and the Actions column respectively.

**Admin View**
Screen Shot 2020-11-20 at 4 03 39 PM

**Tester Permissions View**
Screen Shot 2020-11-20 at 4 18 12 PM

### CODE REVIEW

File: client/components/ConfigureTechnologyRow/index.jsx [hunk 1]
```diff
@@ -30,13 +30,10 @@ class ConfigureTechnologyRow extends Component {
     }
```
Comment (evmiguel) [inline client/components/ConfigureTechnologyRow/index.jsx:30]: This is linting fix.

File: client/components/SignupInstructions/index.jsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from 'react';
```
Comment (evmiguel) [inline client/components/SignupInstructions/index.jsx:1]: This is a linting fix.

File: server/services/UsersService.js [hunk 3]
```diff
@@ -249,7 +249,7 @@ async function saveUserAndRoles(options) {
             }
```
Comment (evmiguel) [inline server/services/UsersService.js:249]: Linting fix

File: server/controllers/AuthController.js [hunk 4]
```diff
@@ -68,10 +68,9 @@ module.exports = {
                         accessToken: req.session.accessToken,
```
Comment (evmiguel) [inline server/controllers/AuthController.js:68]: Linting fix

**Review by isaacdurazo — APPROVED** [review]

File: client/components/TestQueueRun/index.jsx [hunk 5]
```diff
@@ -427,35 +418,54 @@ class TestQueueRow extends Component {
...
-                      <Button
-                          variant="primary"
-                          href={`/run/${runId}`}
-                          disabled={!currentUserAssigned}
-                          ref={this.startTestingButton}
-                      >
-                          {this.testsCompletedOrInProgressByThisUser ? "Continue testing" : "Start testing"}
-                      </Button>
-                      {admin && this.renderOpenAsDropdown()}
+                        <Button
```
Comment (evmiguel) [inline client/components/TestQueueRun/index.jsx:445]: Linting fix

**Pull request merged**